### ISSUE TITLE
project environment

### ISSUE THREAD

**Issue by LeiiY** [sailordiary/m3t.pytorch#1]
using  pip install -r requirement.txt，got this error
`Could not find a version that satisfies the requirement torch>=1.1 (from pytorch-lightning->-r requirements.txt (line 1)) (from ver
sions: 0.1.2, 0.1.2.post1, 0.1.2.post2)
No matching distribution found for torch>=1.1 (from pytorch-lightning->-r requirements.txt (line 1))
`

**Comment by sailordiary** [comment]
Which Python version are you using?
BTW I will be cleaning up the repo folder further this week ; )

**Comment by sailordiary** [comment]
I think you might be using Windows -- is this issue related? https://github.com/pytorch/pytorch/issues/18384

**Comment by LeiiY** [comment]
> I think you might be using Windows -- is this issue related? [pytorch/pytorch#18384](https://github.com/pytorch/pytorch/issues/18384)

yeah，windows10, so ur system version?Ubuntu 16.04 or other

**Comment by sailordiary** [comment]
Yes. You might want to try running on Linux or follow the official instructions to install PyTorch on Windows.

**Comment by LeiiY** [comment]
download the checkpoint
wget 

 to report CCC on the validation set
python3 eval.py --test_on_val --checkpoint m3t_mtl-vox2.pt
python3 get_smoothed_ccc predictions_val.pt

to generate test set predictions
python3 eval.py --checkpoint m3t_mtl-vox2.pt

the weget link？
